sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/CurrencyResolutionPropertiesHelper.java

```diff
@@ -14,7 +14,7 @@ private CurrencyResolutionPropertiesHelper() { }
      * Type defining the methods to set the non-public properties of an {@link CurrencyResolution} instance.
      */
     public interface CurrencyResolutionAccessor {
-        void setISO4217(CurrencyResolution currencyResolution, String iso4217);
+        void setIso4217(CurrencyResolution currencyResolution, String iso4217);
```

Comment:
shouldn't this rename come from regenerating on the swagger (once it's merged)?

Reply:
Once the new changes are merged, the model in the implementation folder will be updated, but this model will still stay the same. Java has two CurrencyResolution models, one explored to the public (which is manually copied and edited which will map to the model in the implementation) and another one hide in the implementation folder (which is autogenerated.